test/quantization/test_quantized_op.py

```diff
@@ -118,68 +118,172 @@ def _get_random_tensor_and_q_params(shapes, rand_scale, torch_type):
 
 class TestQuantizedOps(TestCase):
 
+    # The activation_test_config contains metadata about the quantized activation
+    # functions that will be tested. It allows simpler and more concise unit tests
+    # to be written by specifying the configurations needed for the test in the
+    # activation_test_config and call the provided helper function _test_activation.
+    # Here's a list of the fields that can be included in the config:
+    # quantized_fn: a list of the quantized functions to be tested
+    # reference_fn: the original reference function to be called on the
+    # the dequantized X
+    # inplace_kwarg: the additional inplace keyword argument to test in-place
+    # for each test entry in ops_under_test, it must have at least the fields
+    # for quantized_fn and reference_fn. If inplace_kwarg is missing, the
+    # quantized function is assumed to be either inplace by default or the
+    # test is not testing an inplace function.
+    # output_range: the output range the operator will map to. By default, if it is
+    # no specified, the range will be (-inf, inf)
+    activation_test_config = {
```

Comment:
I feel moving the config to the tests would be better?

Reply:
Do you mean make it part of the helper function below (`_test_activation_function`)? If so, we were originally planning on doing it, but I think it just bloats thee method. Also, we were thinking in the future we can separate the activations functions into separate class, say `TestQuantizedActivationsOps`, which would be configuration based only. In leu of the that, we kinda converged to the class-level config. Do you think we should move it inside the helper method still?

Reply:
I mean put this in individual tests, like `test_qrelu`, that way it is easier to see what is been tested from the test itself

Reply:
I think putting in individual tests would mean one less lookup, not sure what is the benefit of putting this in class level?

Reply:
That makes sense. By distributing the config individually into each of the test function, they can be changed more easily and directly. I will add that change!